src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -448,35 +424,6 @@ PARAM_DEFINE_FLOAT(FW_LND_HVIRT, 10.0f);
  */
 PARAM_DEFINE_FLOAT(FW_LND_FLALT, 3.0f);
```

Comment:
3 seems a bit high. Now that we also have the time*vz based threshold I would reduce that one and more rely on the time based one. 
E.g. 2?

Reply:
And add also a comment here about the max() of the two is taken.

Reply:
reduced, documented [dec33ca](https://github.com/PX4/PX4-Autopilot/pull/19871/commits/dec33ca69ea7f98093d0c3880343330849d58560)